atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -60,7 +60,15 @@ def run(self):
 
         target_info = self.xmlrpc.getBuildTarget(self.target)
         self.log.debug('target info: %s', target_info)
+        if not target_info:
+            raise RuntimeError("no such tag: %s" % self.target)
+
         tag_id = target_info['dest_tag']
+        try:
+            assert self.xmlrpc.checkTagPackage(tag_id, component)
```

Comment:
Any reason to do this rather than just `if not self.xmlrpc.checkTagPackage(): raise ...`?

Didn't know about checkTagPackage, nice find.